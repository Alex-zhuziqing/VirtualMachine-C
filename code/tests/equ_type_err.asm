FUNC LABEL 0
    MOV STK A VAL 8
    EQU STK A
    RET
