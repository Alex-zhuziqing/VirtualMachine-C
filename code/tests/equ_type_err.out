EQU operation can not be applied to a non-REGISTER type
